docs/operator-guide/troubleshooting/resource-observability.md

```diff
@@ -0,0 +1,37 @@
+# Resource Observability Issue
+
+## Problem
+
+Users can't see resources within the KubeRocketCI dedicated namespace via KubeRocketCI portal.
+
+## Cause
+
+The problem might be caused by several factors. First of all, default and allowed namespaces may be either usnset or set incorrectly. Secondly, service account used for browsing the KubeRocketCI portal may have insufficient permission set.
+
+## Solution
+
+1. Check both default and allowed namespace by navigating to the **KubeRocketCI portal** -> **Settings** -> **Cluster Settings**.
+
+2. Check access rights of the service account you use using the kubectl describe command: check sa
+
+  ```bash
```

Comment:
In my opinion, this is the wrong way to solve this issue.

- We need to check which Keycloak groups the user belongs to.
- We need to check what permissions the Keycloak groups provide.
For this, we need to find role bindings and check the roles in them."